modules/shared/logic/votesLogic/votesLogic.test.ts

```diff
@@ -0,0 +1,43 @@
+import { calcProgress, calcTotalVotes, findMinMaxVotes } from './votesLogic';
+
+const firstVote = 80;
+const secondVote = 70;
+const thirdVote = 40;
+const forthVote = 60;
+
+describe('Testing voting stats logic', () => {
+  it('calcTotalVotes should return number of total votes', () => {
+    const votesGroup: number[] = [firstVote, secondVote, thirdVote, forthVote];
+
+    const totalVots: number = calcTotalVotes(votesGroup);
+
+    const results = 250;
+
+    expect(totalVots).toEqual(results);
+  });
+
+  it('calcProgress should return number of progress', () => {
+    const votes = 50;
+    const totalVotes = 200;
+
+    const progress: number = calcProgress(votes, totalVotes);
+
+    const results = 25;
+
+    expect(progress).toEqual(results);
+  });
+
+  it('findMinMaxVotes should return array of max and min votes', () => {
```

Comment:
I would suggest simplifying this test as follow (please watch the recorded video for more details):

```js 

  it('findMinMaxVotes should return array of max and min votes', () => {
    const firstVote = 80;
    const secondVote = 70;
    const thirdVote = 40;
    const forthVote = 60;
    const votesGroup = [firstVote, secondVote, thirdVote, forthVote];

    const [minVotes, maxVotes] = findMinMaxVotes(votesGroup);

    expect(minVotes).toEqual(thirdVote);
    expect(maxVotes).toEqual(firstVote);
  });
    
    ```